casper/src/test/scala/coop/rchain/casper/ValidateTest.scala

```diff
@@ -792,6 +792,52 @@ class ValidateTest
       } yield result
   }
 
+  "Block validDeploySignatures" should "return Ok if all deploys are signed correctly" in withStorage {
+    implicit blockStorey =>
+      implicit blockDagStorage =>
+        // given
+        val (sk, pk) = Ed25519.newKeyPair
+        val block    = MultiParentCasperTestUtil.createGenesis(Map(pk -> 1))
```

Comment:
Why is building the heavy genesis block needed to test such a simple scenario?

Reply:
I've used existing test API that requires the genesis block to create block

```
trait BlockGenerator {
(...)
  def createBlock[F[_]: Monad: Time: BlockStore: IndexedBlockDagStorage](
      parentsHashList: Seq[BlockHash],
      genesis: BlockMessage,
     (...)

```